patches/@types+draft-js+0.11.9.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/node_modules/@types/draft-js/index.d.ts b/node_modules/@types/draft-js/index.d.ts
+index ac2e584..3aa03ed 100755
+--- a/node_modules/@types/draft-js/index.d.ts
++++ b/node_modules/@types/draft-js/index.d.ts
+@@ -419,7 +419,21 @@ declare namespace Draft {
+             /**
+              * Valid inline styles.
+              */
+-            type DraftInlineStyleType = 'BOLD' | 'CODE' | 'ITALIC' | 'STRIKETHROUGH' | 'UNDERLINE';
++            type DraftInlineStyleType =
++              | 'BOLD'
++              | 'CODE'
++              | 'ITALIC'
++              | 'STRIKETHROUGH'
++              | 'UNDERLINE'
++              | 'NONE'
++              | 'CUSTOM-ITALIC'
++              | 'CUSTOM-UNDERLINE'
++              | 'CUSTOM-WEIGHT'
++              | 'CUSTOM-WEIGHT-700'
++              | 'CUSTOM-WEIGHT-900'
++              | 'CUSTOM-COLOR'
++              | 'CUSTOM-LETTERSPACING'
```

Comment:
It's not limited to these weights either, though. And the custom color and letter spacing can have any of a number of valid prefixes like `CUSTOM-COLOR-BADA55` (IIRC).

And also, custom style types are supported by this package, why do we have to add those types inside that? Well, I guess it's because the types have been tagged on by third-party developers later.

Reply:
This is the only way I've found to be able to extend `DraftInlineStyleType` with our custom types. There's probably a better way, so if you know how to make this work, please let me know.